docs/input/docs/reference/custom-formatting.md

```diff
@@ -82,6 +82,30 @@ assembly-informational-format: "{Major}.{Minor}.{Patch}-{CommitsSinceVersionSour
 assembly-informational-format: "{SemVer}-{BranchName:l}"
 ```
 
+## Legacy .NET Composite Format Syntax
+
+GitVersion maintains backward compatibility with legacy .NET composite format syntax using semicolons for positive/negative/zero sections:
+
+```yaml
+# Legacy zero-padded with empty fallback
+assembly-informational-format: "{Major}.{Minor}.{Patch}-{CommitsSinceVersionSource:0000;;''}"
+# Result: "6.13.54-0002" (or "6.13.54" when CommitsSinceVersionSource is 0)
+
+# Three-section format: positive;negative;zero
+assembly-informational-format: "{Value:positive;negative;zero}"
+
+# Two-section format: positive;negative
+assembly-informational-format: "{Value:pos;neg}"
```

Comment:
Could these last two examples be modified to be more practical and real-world relevant, with a line showing the result, as in the first code example?